.gitignore

```diff
@@ -57,3 +57,4 @@ docs/_build/
 
 # PyBuilder
 target/
+.idea/
```

Comment:
This is specific to your IDE. It belongs to [your global .gitignore](https://docs.github.com/en/get-started/git-basics/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer), not in this project.